### DISCUSSION TITLE
X-robots-tag configuration

### DISCUSSION THREAD

**Discussion by BeltserG** [vercel/community#4774] [Help]
### Summary

Hello!
I have an issue: search engines don't see my website. I am using production deployment with custom domain name.
With curl check on my domain a have x-robots-tag : noindex
How can i configure x-robots-tag for search engines?

### Steps to Reproduce

![x-tag](https://github.com/vercel/community/assets/118733491/353f1559-3833-4027-ad6d-cee7f0578a0c)

**Comment by BeltserG** [comment]
Closed

**Reply by frangelli** [reply]
Did you solve your issue? I'm having the same issue.
